refarch-backend/src/test/java/de/muenchen/refarch/configuration/nfcconverter/NfcHelperTest.java

```diff
@@ -82,6 +82,7 @@ void nfcConverterCookie() {
         assertEquals(THIRD_NFC, nfcCookie.getPath());
     }
 
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

Comment:
Is pmd here buggy or why needed?

Reply:
Yes PMD does not recognize that asserts are available, because those are execute inside the forEach.